packages/aws-cdk/README.md

```diff
@@ -560,6 +561,114 @@ This feature currently has the following limitations:
+with the same resource configuration as the provided template.json.
+
+```console
+├── README.md
+├── bin
+│   └── my_awesome_application.ts
+├── cdk.json
+├── jest.config.js
+├── lib
+│   └── my_awesome_application-stack.ts
+├── package.json
+├── tsconfig.json
+```
+
+#### Generate a python application from a deployed stack
+
+If you already had a CloudFormation stack deployed in your account and would like to migrate to using CDK instead of CloudFormation.
+You can use the `--from-stack` option to generate the application. In this case the `--stack-name` must match the name of the deployed stack. 
+
+```console
+$ # generate a python application from MyDeployedStack in your acount
+$ cdk bootstrap migrate --stack-name MyDeployedStack --language python --from-stack
+```
+
+This will generate a Python CDK application which will synthesize to the same configuration of resources as the deployed stack.
+
+#### **CDK Migrate Limitations**
+
+| Situation                                                                     | Result                                                                                            |
+| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Provided Template + stack-name is from a deployed stack in the account/region                     | The CDK application will deploy as the existing stack                         |
+| Provided Template has no overlap with resources already in the account/region                     | The CDK application will deploy a new stack successfully                      |
+| Provided Template has overlap with Cloudformation managed resources already in the account/region | The CDK application will not be deployable unless those resources are removed |
+| Provided Template has overlap with unmanaged resources already in the account/region              | The CDK application will not be deployable until those resources are adopted with CDK Import |
+
+CDK Migrate does not currently support nested stacks, Custom resources, or the `Fn::ForEach` intrinsic function.
+CDK Migrate will only generate L1 constructs and does not currently support any higher level abstractions.
+CDK Migrate succesfully generating an application does *not* guarantee the application is immediately deployable.
+It simply generates a CDK application which will synthesize a template that has identical resource configurations 
+to the provided template. CDK Migrate does not interact with the CloudFormation service to verify the template 
+provided can deploy on its own, nor does it validate that any resources in the provided template are already managed 
+in other CloudFormation Templates. In practice this is how CDK Migrate generated applications will 
+operate in the following scenarios:
```

Comment:
Maybe also call out that if the template has has parameters without default values, the customer will need to fill those in manually